### ISSUE TITLE
why does the exe need java on target machine to run? 

### ISSUE THREAD

**Issue by uneedsec** [tiagorlampert/sAINT#1]
cant it run without java to make it less suspicious? because tje victim now has to install java before it will work

**Comment by tiagorlampert** [comment]
The JRE is required, the executable will not work without it. Because the Windows need JRE for translating the program from java byte code to machine language.

In some cases it may become unviable the use for this reason.

_I will add this question to the FAQ in readme._

**Comment by uneedsec** [comment]
is there no way to make this a standalone executable without prompting the user to install java before it works?

**Comment by tiagorlampert** [comment]
No, it is necessary for the user to have Java installed on the target system.
